### PR TITLE
Polish ✨ 

### PR DESCRIPTION
**Pull request by daiyiwhale**
Polish based on designer feedback! 💜 

Improvements in this PR:
- Removed `+` in `+ New [Thing]` buttons for consistency across all buttons
- FilterHeadline now pluralises item names when appropriate (e.g. 'Showing 1 Component' and 'Showing 2 Components')
- Removed "Services" headline when no Services to display
- Hide the divider line in `PageHeader` when there are no tabs
- Fixed column display bug in `TaskTable`
- Fixed negative margin in `PageHeader` action buttons
- Moved `FilterInputText` (search boxes) to the leftmost in `FilterBar` (this is going to look weird to us for a while (': )

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @kennyt

**Comment by rcorral** [comment]
retest this please

**Comment by rcorral** [comment]
I think there's legitimate tests failing this time, integration tests.

File: src/js/components/UnitHealthSidePanelContents.js [hunk 1]
```diff
@@ -1,179 +0,0 @@
-/* eslint-disable no-unused-vars */
```
Comment (kennyt) [inline src/js/components/UnitHealthSidePanelContents.js]: 💵

**Comment by MatApple** [comment]
flicts

**Comment by daiyiwhale** [comment]
Conflicts addressed!

File: src/js/components/TaskView.js [hunk 2]
```diff
@@ -160,23 +160,24 @@ class TaskView extends mixin(SaveStateMixin) {
         <FilterHeadline
           inverseStyle={inverseStyle}
           onReset={this.resetFilter}
-          name="Tasks"
+          name="Task"
           currentLength={tasks.length}
           totalLength={totalNumberOfTasks} />
         <FilterBar>
+          <div className="form-group flush-bottom">
```
Comment (MatApple) [inline src/js/components/TaskView.js]: Should we still have this wrapper?

File: src/js/components/HealthTab.js [hunk 3]
```diff
@@ -132,7 +132,7 @@ class HealthTab extends React.Component {
         <FilterHeadline
           currentLength={visibleData.length}
           inverseStyle={true}
-          name={"Health Checks"}
+          name={"Health Check"}
```
Comment (jfurrow) [inline src/js/components/HealthTab.js]: Should we pluralize this based off of `units.getItems().length` (the total number of health checks)?

**Comment by MatApple** [comment]
Are we able to delete all `SidePanel` components now?? If so we should do that here too :)

File: src/js/components/TaskView.js [hunk 4]
```diff
@@ -160,23 +160,24 @@ class TaskView extends mixin(SaveStateMixin) {
         <FilterHeadline
           inverseStyle={inverseStyle}
           onReset={this.resetFilter}
-          name="Tasks"
+          name="Task"
```
Comment (jfurrow) [inline src/js/components/TaskView.js]: We should probably pluralize this too, based off the total number of tasks (and not the filtered number of tasks)

File: src/js/pages/NodesPage.js [hunk 5]
```diff
@@ -251,10 +251,11 @@ var NodesPage = React.createClass({
         <FilterHeadline
           inverseStyle={true}
           onReset={this.resetFilter}
-          name="Nodes"
+          name="Node"
```
Comment (jfurrow) [inline src/js/pages/NodesPage.js]: Pluralize based on `data.totalNodes`?

**Comment by jfurrow** [comment]
After Mat's comments, :ship:!

**Comment by daiyiwhale** [comment]
@MatApple I believe that is a separate thing! There's some weird thing with the dashboard. I will investigate in a new PR (:

File: src/js/components/TaskView.js (hunk 2 above)
Comment (daiyiwhale) [inline src/js/components/TaskView.js]: good catch (: removed!

**Comment by MatApple** [comment]
Sounds good @daiyiwhale. We've got conflicts now.. :( Will Merge once resolved!

**Comment by daiyiwhale** [comment]
Done!

**Pull request merged**